core/controllers/contributor_dashboard_test.py

```diff
@@ -1397,26 +1397,66 @@ class FeaturedTranslationLanguagesHandlerTest(test_utils.GenericTestBase):
     """Test for the FeaturedTranslationLanguagesHandler."""
 
     def test_get_featured_translation_languages(self) -> None:
-        response = self.get_json('/retrivefeaturedtranslationlanguages')
-        self.assertEqual(
-            response,
-            {'featured_translation_languages': []}
-        )
-
-        new_value = [
-            {'language_code': 'en', 'explanation': 'Partnership with ABC'}
-        ]
-        config_services.set_property(
-            'admin',
-            'featured_translation_languages',
-            new_value
-        )
+        response = self.get_json('/retrievefeaturedtranslationlanguages')
+        expected_response = {
+            'featured_translation_languages': [
+                {
+                    'language_code': 'pt',
+                    'explanation': 'For learners in Brazil, Angola '
+                    'and Mozambique.'
+                },
+                {
+                    'language_code': 'ar',
+                    'explanation': 'For learners in Arabic-speaking countries '
+                    'in the Middle East.'
+                },
+                {
+                    'language_code': 'pcm',
+                    'explanation': 'For learners in Nigeria.'
+                },
+                {
+                    'language_code': 'es',
+                    'explanation': 'For learners in Latin America and South '
+                    'America.'
+                },
+                {
+                    'language_code': 'sw',
+                    'explanation': 'For learners in Kenya and Tanzania.'
+                },
+                {
+                    'language_code': 'hi',
+                    'explanation': 'For learners in India'
+                },
+                {
+                    'language_code': 'ha',
+                    'explanation': 'For learners in Nigeria.'
+                },
+                {
+                    'language_code': 'ig',
+                    'explanation': 'For learners in Nigeria.'
+                },
+                {
+                    'language_code': 'yo',
+                    'explanation': 'For learners in Nigeria.'
+                }]
+        }
+        self.assertEqual(response, expected_response)
```

Comment:
Can we use `constants.FEATURED_TRANSLATION_LANGUAGES` instead?

Reply:
Well, we can but previously Sean mentioned that in tests we should hardcode the value to check the output if it matches or not.
Let me know what you think.

Thanks